### ISSUE TITLE
.get_file() helper in MakingNewAnnotationPackages.Rnw not working

### ISSUE THREAD

**Issue by hpages** [Bioconductor/AnnotationForge#32]
[This code](https://github.com/Bioconductor/AnnotationForge/blob/aed6ba064f6eee028c60c9b10cb429019a27d159/vignettes/MakingNewAnnotationPackages.Rnw#L195-L215) in the `MakingNewAnnotationPackages.Rnw` vignette  doesn't seem to work properly anymore in BioC 3.15. In release, `uquery` actually looks like the path to a local file, but in devel it looks something like this:
```
> uquery
[1] "de8e479a7fb8c_%3Fquery%3DP13368%2Bor%2BQ6GZX4%26format%3Dtab"
```
This breaks **AnnotationForge**: https://bioconductor.org/checkResults/3.15/bioc-LATEST/AnnotationForge/

@LiNk-NY Do you think you can take a look at it?

Thanks,
H.

**Comment by LiNk-NY** [comment]
Fixed in 852ed89abd06b6b1eb5414288ebe42959b00b421

**Comment by hpages** [comment]
**AnnotationForge** back to green on 3.15 daily report. Thanks @LiNk-NY !
